source-code/ide-extension/src/main.ts

```diff
@@ -20,6 +20,7 @@ import { openInEditorCommand } from "./commands/openInEditor.js"
 import { editMessageCommand } from "./commands/editMessage.js"
 import { openInlangProject, tryCatch } from "@inlang/app"
 import { createFileSystemMapper } from "./services/inlang-environment/src/createFileSystemMapper.js"
+import { _import } from "./services/inlang-environment/src/_import.js"
```

Comment:
@felixhaeberle i would not classify `_import` as an (external) service. and neither does inlang-environment exist anymore

Reply:
better?